docs/cody/model-configuration/examples.mdx

```diff
@@ -0,0 +1,681 @@
+# Examples
+
+## Sourcegraph-supplied models only
+
+This section includes examples how to configure Cody to use Sourcegraph-supplied models.
+
+-   [Minimal configuration](/cody/model-configuration#configure-sourcegraph-supplied-models)
+-   [Using model filters](/cody/model-configuration#model-filters)
+-   [Change default models](/cody/model-configuration#default-models)
+
+## Sourcegraph-supplied models and BYOK (Bring Your Own Key)
+
+Sourcegraph-supplied models come with preconfigured providers, identified by the following IDs (namespaces):
+
+-   "anthropic"
+-   "google"
+-   "fireworks"
+-   "mistral"
+-   "openai"
+
+### Override provider config for all models in the namespace
+
+When Sourcegraph-supplied models are used and a provider override for a Sourcegraph-supported provider (same ID) is specified,
+the override applies to all Sourcegraph-supplied models within that provider.
+For example, if you specify an override for a provider with ID `"anthropic"`, it will apply to all models from the `"anthropic"` provider.
```

Comment:
Maybe rephrase to something like:

> Instead of Sourcegraph using its own servers to make LLM requests, it is possible to bring your own API keys for a given model provider. For example, if you wish for all Anthropic API requests to go directly to your own Anthropic account and use your own API keys instead of going via Sourcegraph's servers, you could override the `anthropic` provider's configuration:

Reply:
Might also make sense to mention that the change only applies to Anthropic models, and that other models would still go through our servers - or that you could disable other models from being ued via the allow/deny list option.

Reply:
Thank you, I committed the suggested change.

> other models would still go through our servers

It's mentioned in the example description a few lines below.

> you could disable other models from being used via the allow/deny list option

We have a dedicated section for the model filters.